include/envoy/server/instance.h

```diff
@@ -219,6 +219,11 @@ class Instance {
    */
   virtual std::chrono::milliseconds statsFlushInterval() const PURE;
 
+  /**
+   * Flush the server stats
```

Comment:
Can you say more in the doc about how this explicit call interacts with the flush-interval above?

Reply:
updated. Not sure how much more explicit I want to be without talking about the InstanceImpl implementation itself.